### ISSUE TITLE
Crear Wiki QRDA.org.ve

### ISSUE THREAD

**Issue by qr-da** [qr-da/qr-da#1]
Crear la wiki que contendrá toda la información del evento

**Comment by Richzendy** [comment]
Que CMS sugieren? media-wiki ?

O algo acá mismo en github?

**Comment by yuryja** [comment]
Y a mi no me parece mal usar la wiki de github. Qué opinan?

**Comment by davidhdz** [comment]
Sí, yo decía en el evento que empezando podríamos usar las herramientas que nos provee github, para agilizar, dinamizar y no dormirnos esperando que se monte una plataforma tal para hacer tal otra y bla bla bla.

**Comment by zamuro** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Genial. ¿A dónde hay que apuntar los DNS? Para empezar con eso cuanto
antes.

El 17/10/15 a las 08:12, David Hernández escribió:

> Sí, yo decía en el evento que empezando podríamos usar las
> herramientas que nos provee github, para agilizar, dinamizar y no
> dormirnos esperando que se monte una plataforma tal para hacer tal
> otra y bla bla bla.
> 
> — Reply to this email directly or view it on GitHub 
> https://github.com/qr-da/qr-da/issues/1#issuecomment-148914051.

---

MSc. Ing. Manuel Lucena Pérez
Cisco Certified Network Professional®
3Com WAN-LAN Specialist®
GNU/Linux User #409072
Debian User #261
Venezolana de Tecnologías, S.A. - Chief Technical Officer
m: +58 412 5156029
skype: mlucenap
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iQEcBAEBAgAGBQJWI4S4AAoJELoVRjyX+zSbC80H/39Zw7ihNEhediCM7mduIlol
4fIR20cEU8q2QMxH3VziE4iwqDjEII8n1ABdki0aXkpFSaNn8k6T0bPs7/sZnWBf
8wFBquc/F1SqvhNXduPkBPhzP4hkW5dUhBBJp2hZSe29Ju6SY9TdsVD7lg8D3o6p
l+h1mRmexgkd0WQquJvkxvKMj94+vjHWkOzMI8py0b+E5fjsLbLcGaw/O5YjpjCv
25IAeIuL4ZgGnA7L07R17r8ae+guZz0cNRYQyMsuSFnW+xdxVVlB0tEDq1yuCq5K
tbJWF9eqLzYqxLkpHNynAXLyv2vIcd3hr3hOOQfxruDbJ5GrLjcQ7B92oZYRPA4=
=iZ2y
-----END PGP SIGNATURE-----

**Comment by davidhdz** [comment]
Pues no sé si usamos qrda.com.ve mientras se adquiere qrda.org.ve

**Comment by zamuro** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Si la memoria no me falla, ya julioh compró el dominio. Mañana le
pregunto si ya creó los DNS. If so, los apuntamos a gh.

El 18/10/15 a las 07:33, David Hernández escribió:

> Pues no sé si usamos qrda.com.ve mientras se adquiere qrda.org.ve
> 
> — Reply to this email directly or view it on GitHub 
> https://github.com/qr-da/qr-da/issues/1#issuecomment-149005973.

---

MSc. Ing. Manuel Lucena Pérez
Cisco Certified Network Professional®
3Com WAN-LAN Specialist®
GNU/Linux User #409072
Debian User #261
Venezolana de Tecnologías, S.A. - Chief Technical Officer
m: +58 412 5156029
skype: mlucenap
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iQEcBAEBAgAGBQJWI4ruAAoJELoVRjyX+zSbAuwH+gLc1OqEMasUM0XbCzqUNrLG
Y3ERwZXTRAnGWpZr2ofqDOr1bo36u58QtzgGmNjNRsc0J94XfzUc0e3DkG2P81uK
Dfn0RUv7w7qt2z06NQGR4OOeO5zMnz4yEYOa5YN9dD00yqnS2ti1BkbBPSTQ5H2h
Y28pxcqRPvrof76OgbipMkeDoPc3aoK8K40UDTbucRGYvzutUh6XyjAnOdAwiqPK
f6pS8oeCowxmEYj5gc5ca3IF5vqD4Dbnx2JUiBysmPCqEgpnHm557n+vxo8onrtx
qgwplc44Vug9SQxyB0qSL5IVDAYO4nXqkJjm99COpL8Tz6sRV+b8RE7CEb50gpU=
=cf8d
-----END PGP SIGNATURE-----

**Comment by davidhdz** [comment]
Fino chamo, yo revisé esta mañana y qrda.org.ve estaba todavía disponible, pero conociendo como manejan nic.ve es que no han revisado la transacción todavía.

**Comment by talueses** [comment]
Confirmo. Hasta la fecha aún el dominio está disponible. Y secundo la idea usar la wiki de github, por mientras...
